timestampvm/vm.go

```diff
@@ -89,15 +90,15 @@ func (vm *VM) Initialize(
 	_ []*common.Fx,
 	_ common.AppSender,
 ) error {
-	version, err := vm.Version()
+	version, err := vm.Version(ctx)
 	if err != nil {
 		log.Error("error initializing Timestamp VM: %v", err)
 		return err
 	}
 	log.Info("Initializing Timestamp VM", "Version", version)
 
 	vm.dbManager = dbManager
-	vm.ctx = ctx
+	vm.ctx = snowCtx
```

Comment:
I wonder if it makes sense to change the name of the struct as well to reduce confusion

Reply:
You're right